src/Raven.Studio/typescript/viewmodels/manage/storageReport.ts

```diff
@@ -233,7 +233,7 @@ class storageReport extends viewModelBase {
                 []
             );
             
-            item.recyclableJournal = temp.Type === "RecyclableJournal";
+            item.recyclableJournal = false;
```

Comment:
I think we need to make adjustments to storage report now?

Reply:
Yes